src/subdomains/generic/user/services/webhook/dto/payment-webhook.dto.ts

```diff
@@ -24,6 +24,9 @@ export class PaymentWebhookData extends TransactionDto {
 
   @ApiProperty()
   dfxReference: number;
+
+  @ApiProperty()
+  bankAccount: string;
```

Comment:
Maybe use the name `targetAccount` (or something similar) and set it to the blockchain address for BuyCrypto?